manager-bundle/src/Command/MaintenanceModeCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\ManagerBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+use Twig\Environment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Error\SyntaxError;
+use Webmozart\PathUtil\Path;
+
+/**
+ * @internal
+ */
+class MaintenanceModeCommand extends Command
+{
+    private string $webDir;
+    private Environment $twig;
+    private Filesystem $filesystem;
+
+    public function __construct(string $webDir, Environment $twig, Filesystem $filesystem = null)
+    {
+        $this->webDir = $webDir;
+        $this->twig = $twig;
+        $this->filesystem = $filesystem ?? new Filesystem();
+
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('contao:maintenance-mode')
+            ->addArgument('state', InputArgument::REQUIRED, 'Use "enable" to enable and "disable" to disable the maintenance mode. If the state is already the desired one, nothing happens. You can also use "on" and "off".')
+            ->addOption('template', 't', InputOption::VALUE_REQUIRED, 'Allows to take a different Twig template name when enabling the maintenance mode.', '@ContaoCore/Error/service_unavailable.html.twig')
```

Comment:
I wonder if we should add a new `maintenance.html.twig` template to the Contao template folder and use `@Contao/maintenance.html.twig` here. Slowly moving away from bundle templates.

Could be done in a later PR as well.

Reply:
I don't know the exact implications but it's your area of expertise :) I chose the existing one because people might have already adjusted it. That means they have the major benefit of not having to change a thing at all :)
So if we want to change anything here, we should probably do that later, yes.

Reply:
On my list.